src/components/SourceItem/SourceItem.tsx

```diff
@@ -25,13 +25,19 @@ export const SourceItem: React.FC<SourceProps> = ({ source }) => {
 
   return (
     <Paper elevation={1} sx={{ overflow: 'hidden' }}>
-      <Grid container direction="column" spacing={2}>
+      <Grid
+        container
+        direction="column"
+        spacing={2}
+        sx={{ position: 'relative' }}
+      >
         <Grid
           item
           sx={{
             position: 'absolute',
             transform: 'translate( -50%, -35%)',
             left: '49%',
+            overflow: 'visible',
```

Comment:
I played around with this for a bit and couldn't figure it out either, this needs some CSS magic :)
Just arrange the favicon within the bounding box for now and write an issue to modify it later so it looks like in the UI design.